### PR TITLE
 Update Osram Lightify documentation to reflect the latest changes

### PR DESCRIPTION
**Pull request by OleksandrBerchenko**
**Description:**

1. Update Osram Lightify documentation to reflect the latest changes.
2. Fix a broken link to Osram web-site.

**Pull requests in [home-assistant](https://github.com/home-assistant/home-assistant):** https://github.com/home-assistant/home-assistant/pull/22184 and https://github.com/home-assistant/home-assistant/pull/22597

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/light.osramlightify.markdown [hunk 1]
```diff
@@ -12,13 +12,13 @@ ha_category: Light
...
 
-The `osramlightify` platform allows you to integrate your [Osram Lightify](http://www.osram.com/osram_com/products/led-technology/lightify/index.jsp) into Home Assistant.
+The `osramlightify` platform allows you to integrate your [Osram Lightify](https://www.osram.com/cb/lightify/index.jsp) into Home Assistant.
 
 ```yaml
 # Example configuration.yaml entry
 light:
   - platform: osramlightify
-    host: 192.168.0.50
+    host: 192.168.1.50
```
Comment (klaasnicolaas) [inline source/_components/light.osramlightify.markdown]:
```suggestion
    host: IP_ADDRESS
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/light.osramlightify.markdown [hunk 2]
```diff
@@ -27,13 +27,57 @@ host:
   required: true
   type: string
 allow_lightify_nodes:
-  description: (true/false) Edit this to stop homeassistant from importing the lightify lights.
+  description: (true/false) Should Home Assistant import individual lights?
```
Comment (klaasnicolaas) [inline source/_components/light.osramlightify.markdown]: Would rather make something that indicates: if you do `true` then this happens and if you do `false` then that happens....

File: source/_components/light.osramlightify.markdown [hunk 3]
```diff
@@ -27,13 +27,57 @@ host:
...
+  default: true
+  type: boolean
+allow_lightify_switches:
+  description: (true/false) Should Home Assistant import switches? Takes effect only if `allow_lightify_nodes` is `true`.
   required: false
   default: true
   type: boolean
 allow_lightify_groups:
-  description: (true/false) Edit this to stop homeassistant from importing the lightify groups.
+  description: (true/false) Should Home Assistant import groups?
```
Comment (klaasnicolaas) [inline source/_components/light.osramlightify.markdown]: same

File: source/_components/light.osramlightify.markdown (hunk 2 above)
Comment (OleksandrBerchenko) [reply source/_components/light.osramlightify.markdown]: Done.

File: source/_components/light.osramlightify.markdown (hunk 3 above)
Comment (OleksandrBerchenko) [reply source/_components/light.osramlightify.markdown]: Done.

**Comment by OleksandrBerchenko** [comment]
@klaasnicolaas Both PRs to Home Assistant repo have been merged already!

**Review by klaasnicolaas — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
Well then it is time to merge @OleksandrBerchenko 😉

**Pull request merged**